hello-world/wrapper/typescript/shims/shims.js

```diff
@@ -0,0 +1,251 @@
+const textEncoderShims = require("./textEncoder");
+const msgpackShims = require("./msgpack");
+
+const packagesShim = `
```

Comment:
For jetbrains IDEs, it's helpful to annotate strings that contain code with a comment indicating the language:

```ts
// language=javascript
const packagesShim = `
...
```

This provides syntax highlighting etc.

If it doesn't hurt anything for vs code users, could we add the comment here?